TombEngine/Game/effects/effects.cpp

```diff
@@ -1271,7 +1271,7 @@ void TriggerFlashSmoke(int x, int y, int z, short roomNumber)
 	spark->mirror = mirror;
 }
 
-void TriggerFireFlame(int x, int y, int z, FlameType type)
+void TriggerFireFlame(int x, int y, int z, FlameType type, Vector3 color1, Vector3 color2)
```

Comment:
Good opportunity to use `const Vector3&` here for both parameters.

Reply:
fixed